src/Files.App/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -183,34 +190,37 @@
 						<ComboBoxItem Content="{helpers:ResourceString Name=Skip}" />
 					</ComboBox.Items>
 				</ComboBox>
+
 			</Grid>
 
+			<!--  Description  -->
 			<TextBlock
-				x:Name="DescriptionText"
+				x:Name="DescriptionTextBlock"
 				Foreground="{ThemeResource SystemFillColorCautionBrush}"
 				Text="{x:Bind ViewModel.Description, Mode=OneWay}"
 				TextWrapping="WrapWholeWords" />
 
+			<!--  (Divider)  -->
 			<Border
-				x:Name="SeparatorLine"
 				Height="1"
 				HorizontalAlignment="Stretch"
-				Background="{ThemeResource SystemBaseLowColor}"
-				Opacity="0.8"
-				Visibility="{x:Bind ViewModel.FileSystemDialogMode.ConflictsExist}" />
+				Background="{ThemeResource DividerStrokeColorDefaultBrush}"
```

Comment:
Background or Border? I suppuse it won't matter much..

Reply:
What do you mean?

I changed separator control from `MenuflyoutSeparator` to `Border` and background from `SystemBaseLowColor` to `DividerStrokeColorDefaultBrush`